test/e2e/Apps/BasicDotNetIsolated/ClassBasedEntities.cs

```diff
@@ -0,0 +1,82 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Azure.Storage.Blobs;
+using Microsoft.Azure.Functions.Worker;
+using Microsoft.Azure.Functions.Worker.Http;
+using Microsoft.DurableTask;
+using Microsoft.DurableTask.Client;
+using Microsoft.DurableTask.Entities;
+using Microsoft.Extensions.Configuration;
+
+namespace Microsoft.Azure.Durable.Tests.E2E;
+
+public class ClassBasedEntities
+{
+    // HTTP trigger to start the orchestration and wait for it to complete.
+    [Function(nameof(StartClassBasedEntityOrchestration))]
+    public static async Task<HttpResponseData> StartClassBasedEntityOrchestration(
```

Comment:
Any particular reason we defined a new HTTP starter function rather than using the generic StartOrchestration starter?

Reply:
Ah, good catch.